src/framework/components/camera/camera_system.js

```diff
@@ -46,6 +46,11 @@ pc.extend(pc, function () {
             type: "boolean",
             defaultValue: true
         }, {
+            name: "frustumCulling",
+            displayName: "Frustum culling",
+            type: "boolean",
+            defaultValue: false
+        }, {
```

Comment:
I do as some point, but for the purpose of just getting this merged, it simplifies things. If this is exposed, we need to update Editor too. Let's do that later when the new editor is out.